test/functional/suite_test.go

```diff
@@ -91,6 +92,9 @@ var _ = BeforeSuite(func() {
 		Expect(err).NotTo(HaveOccurred())
 		// +kubebuilder:scaffold:scheme
 
+		err = confv1.AddToScheme(scheme.Scheme)
```

Comment:
Minor suggestion, we could keep this as `configv1` like other uses and also use `Install` as `AddToScheme` is deprecated.

```suggestion
		err = configv1.Install(scheme.Scheme)
```

Reply:
Done. Thank you